MaxText/configs/models/llama4-17b-16e.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+decoder_block: "llama4"
+mlp_activations: ["silu","linear"]
+enable_dropout: False
+logits_via_embedding: False
+tokenizer_type: "huggingface"
+tokenizer_path: "meta-llama/Llama-4-Scout-17B-16E"
+
+base_emb_dim: 5120
+base_num_decoder_layers: 48
+base_num_query_heads: 40
+base_num_kv_heads: 8
+vocab_size: 202048
+multiple_of: 2048
+ffn_exp: 4
+normalization_layer_epsilon: 1e-05
+rope_max_timescale: 500000
+rope_type: "llama3.1"
+num_experts: 16
+capacity_factor: -1.0
+auto_scale_F: True
+shared_experts: 1
+num_experts_per_tok: 1
+use_qk_norm: True
+ffn_dim_multiplier: 1.2
```

Comment:
Same comment, let's use the same convention as Llama 2 or Llama3, e.g. no need for the multiplier.